client/post-editor/style.scss

```diff
@@ -61,6 +61,32 @@
 	}
 }
 
+.editor__site {
+	display:  flex;
+	background: lighten( $gray, 30% );
+
+	@include breakpoint( ">660px" ) {
+		display: none;
+	}
+
+	.site__title,
+	.site__domain {
+		&::after {
+			@include long-content-fade( $color: lighten( $gray, 30% ) );
+		}
+	}
+
+	.site__home {
```

Comment:
That's fine for me, I've removed it. I've removed the home icon on mobile in every case, the only other case is in this screen:
![sin titulo](https://cloud.githubusercontent.com/assets/1715800/14936564/522ae796-0ef0-11e6-8bd0-31a9db70dc00.png)

I think that in that case is also more important to see the site icon than a Home icon. I *think* that users on mobile are used to this kind of UI, rows of date where each row is tappable. Plus, just seeing a Home icon doesn't really transmit to me that the thing is tappable... I don't know, you sure know more about this than me, so tell me if it's ok.